src/samples/msal-web-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+aad.clientId=
+aad.authority=
+aad.secretKey=
+aad.redirectUri=
```

Comment:
As discussed, might make sense to include default redirectUri

Reply:
added